modules/020-deckhouse/docs/README_RU.md

```diff
@@ -24,3 +24,27 @@ search: releaseChannel, стабилизация релизного канала
 - **Сервис валидирования custom resource'ов**
 
   Сервис валидирования предотвращает создание custom resource'ов с некорректными данными или внесение таких данных в уже существующие custom resource'ы. Отслеживаются только custom resource'ы, находящиеся под управлением модулей Deckhouse.
+
+
+### DeckhouseReleases
+Список последних релизов в кластере можно получить командной `kubectl get DeckhouseRelease`. По-умолчанию хранится 10 последних релизов и все будущие
+Каждый релиз может иметь один из следующих статусов:
+`Pending` - релиз находится в ожидании, ждет окна обновления, настроек канареечного развертывания и тд. Более подробно можно увидеть с помощь `kubectl describe DeckhouseRelease $name`
+`Deployed` - релиз применен. Это значит, что образ deckhouse уже поменялся на новую версию,
+ но при этом процесс обновления всех компонентов кластера идет асинхронно, так как зависит от многих настроек.
+`Outdated` - релиз устарел и больше не используется
+`Suspended` - релиз был отменен (например в нем обнаружилась ошибка), релиз переходит в этот статус, если его отменили и при этом он еще был применен в кластере
+
+---
```

Comment:
In Deckhouse, there are no separators for paragraphs in the doc. Let's get rid of them. It feels new paragraphs are enough to emphasize the parts of the doc are separated.